site/utils/siri.ts

```diff
@@ -127,7 +127,16 @@ export const getPtSituationElementFromDraft = (disruption: Disruption) => {
                           Severity: consequence.disruptionSeverity,
                           Affects: {
                               ...(consequence.consequenceType === "networkWide" ||
-                              consequence.consequenceType === "operatorWide"
+                              consequence.consequenceType === "stops"
+                                  ? {
+                                        Operators: {
+                                            AllOperators: "",
```

Comment:
when it said all operators in the ticket did it not mean doing this for every operator - please clarify with steven or chris?  
`<AffectedOperator>
    <OperatorRef>FMAN</OperatorRef>
    <OperatorName>First Greater Manchester</OperatorName>
</AffectedOperator>`